Jenkinsfile

```diff
@@ -17,7 +17,7 @@ node {
         }
     }
   } finally {
-    junit 'build/reports/**/*.xml'
+    junit '/target/surefire-reports/*.xml'
```

Comment:
I think leading with the `/` is going to be a problem.  However, I'll run it and see what happens.